src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Controllers/BankIdApiController.cs

```diff
@@ -262,5 +262,17 @@ private static string AppendQueryString(string url, string queryString)
             var delimiter = url.Contains("?") ? "&" : "?";
             return $"{url}{delimiter}{queryString}";
         }
+
+        [ValidateAntiForgeryToken]
+        [HttpPost("Cancel")]
+        public async Task<CancelResponse> Cancel(BankIdCancelApiStatusRequest request)
```

Comment:
There should be a `BankIdLoginApiCancelResponse` to follow current conventions, even if only empty. Don't want to expose the API model to the client.

Reply:
Nice catch!